docs/UserGuide.adoc

```diff
@@ -445,36 +467,71 @@ money without having money in your wallet.
 [width="100%",cols="5%,95%", grid=none]
 |===
 ifdef::env-github[| :white_check_mark: a| You will see a success message and the amount of money you have saved in your savings account.]
-ifndef::env-github[a| icon:check[role="green", size="2x"] a| You will see a success message and the amount of money you have saved in your savings account.]
-|===
+ifndef::env-github[a| icon:check[role="green", size="2x"] a| You will see a success message and the amount of money you have saved in your savings account. You will be able to view your savings in the savings history tab. The savings will appear GREEN.]
 
-[NOTE]
-If you click the "Savings" tab on the right panel, you will be able to view your savings history.
-Also the amount saved is deducted from your wallet.
+.Example savings display when "Savings" is clicked as shown after savings are made.
+image::SavingsAdded.png[]
 
-==== Withdraws money from savings account: `withdraw`
-Allows users to log an amount of money to withdraw from his savings account into his wallet.
-//end::savings[]
+.Example success message that appears after keying a successful saving.
+image::SavingsSuccessMessage.png[]
+
+{empty}
+|===
+
+==== Withdraw money from user's savings account into wallet: `withdraw`
+The user can withdraw a sum of from his/her savings account, and add it into his/her wallet.
 
-//tag::withdrawals[]
 ****
-*Format*: `withdraw` +
+*Format*: `withdraw AMOUNT` +
 *Example*: `withdraw 10`
+
+*Note*: +
+`AMOUNT` must be a non-negative integer or a non-negative double with 2 decimal places.
+
 ****
 
 [IMPORTANT]
-You must have money in your savings account before you can withdraw from it.
+You must have money in your savings account before you can withdraw from it. +
+Also, you cannot withdraw an amount that will result in your wallet having more than $1,000,000.
 
 [width="100%",cols="5%,95%", grid=none]
 |===
-ifdef::env-github[| :white_check_mark: a| You will see a success message and the amount of money you have withdrew from your savings account.]
-ifndef::env-github[a| icon:check[role="green", size="2x"] a| You will see a success message and the amount of money you have withdrew from your savings account.]
+ifdef::env-github[| :white_check_mark: a| You will see a success message and the amount of money you have saved in your savings account.]
+ifndef::env-github[a| icon:check[role="green", size="2x"] a| You will see a success message and the amount of money you have saved in your savings account. You will be able to view your withdrawals in the savings history tab. The savings will appear RED. ]
```

Comment:
should be "you have withdrawn in your wallet" instead of "saved in your saving account" i think, and "The withdrawals will appear RED" instead of "The savings will appear RED".